cylc-overview.tex

```diff
@@ -5,21 +5,20 @@ \section{Cylc Overview}
 Cylc Overview} and the Cylc User Guide.}
 
 Cylc (``silk'') is a workflow engine that can manage continuous workflows of
-cycling (repeating) tasks for applications such as weather and environmental
+cycling asks for applications such as weather and environmental
```

Comment:
Damn, it's amazing what mad typos my brain just skips over when I read my own text!